### PR TITLE
Add looking for a challenge copy to the team section

### PR DESCRIPTION
**Pull request by jfranciscosousa**

### CODE REVIEW

File: src/sections/CompanyTeamSection/index.js [hunk 1]
```diff
@@ -20,8 +22,85 @@ const CompanyTeamSection = () => (
...
+      <TeamElement
+        key="francisco"
+        name="Francisco Sousa"
+        position="Software Developer"
+        image={data.francisco.image.fluid}
+      />
+      <TeamElement
+        key="ines"
+        name="Inês Silva"
+        position="Design"
```
Comment (filipebfernandes) [inline src/sections/CompanyTeamSection/index.js]: *Designer

File: src/sections/CompanyTeamSection/index.css [hunk 2]
```diff
@@ -14,3 +14,44 @@
...
+    grid-template-columns: 1fr;
+  }
+}
+
+.challenge {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  padding: $spacing-56;
```
Comment (filipebfernandes) [inline src/sections/CompanyTeamSection/index.css]: Can you confirm this is correct. Looking at figma, the padding it's 40px and 28px mobile

Comment (jfranciscosousa) [reply src/sections/CompanyTeamSection/index.css]: Yeah padding is smaller on mobile. But the 40 spacings are actually 56, Ines said it

Comment (filipebfernandes) [reply src/sections/CompanyTeamSection/index.css]: 👍

**Pull request merged**